OpenRA.Mods.AS/Traits/AirstrikeMaster.cs

```diff
@@ -126,7 +126,7 @@ public override void InitializeSlaveEntry(Actor slave, BaseSpawnerSlaveEntry ent
 		// invokes Attacking()
 		void INotifyAttack.Attacking(Actor self, in Target target, Armament a, Barrel barrel)
 		{
-			if (IsTraitDisabled || IsTraitPaused || !Info.ArmamentNames.Contains(a.Info.Name))
+			if (target.Type == TargetType.Invalid || IsTraitDisabled || IsTraitPaused || (Info.ArmamentNames.Count > 0 && !Info.ArmamentNames.Contains(a.Info.Name)))
```

Comment:
Add `target.Type == TargetType.Invalid` check here is to fix the crash happens when the master kill the target as soon as it make the slave to attack.

Reply:
The situation above will happen if you replace the dummy weapon of aircraft carrier to a laser cannon.